boards/arm/phy62xx/phy6222/configs/newostest/defconfig

```diff
@@ -0,0 +1,139 @@
+# CONFIG_NSH_DISABLE_BASENAME is not set
+# CONFIG_NSH_DISABLE_CAT is not set
+# CONFIG_NSH_DISABLE_CD is not set
+# CONFIG_NSH_DISABLE_CMP is not set
+# CONFIG_NSH_DISABLE_CP is not set
+# CONFIG_NSH_DISABLE_DF is not set
+# CONFIG_NSH_DISABLE_DIRNAME is not set
+# CONFIG_NSH_DISABLE_ECHO is not set
+# CONFIG_NSH_DISABLE_ENV is not set
+# CONFIG_NSH_DISABLE_EXEC is not set
+# CONFIG_NSH_DISABLE_EXIT is not set
+# CONFIG_NSH_DISABLE_EXPORT is not set
+# CONFIG_NSH_DISABLE_FREE is not set
+# CONFIG_NSH_DISABLE_HELP is not set
+# CONFIG_NSH_DISABLE_HEXDUMP is not set
+# CONFIG_NSH_DISABLE_KILL is not set
+# CONFIG_NSH_DISABLE_LS is not set
+# CONFIG_NSH_DISABLE_MKDIR is not set
+# CONFIG_NSH_DISABLE_MOUNT is not set
+# CONFIG_NSH_DISABLE_PS is not set
+# CONFIG_NSH_DISABLE_PSSTACKUSAGE is not set
+# CONFIG_NSH_DISABLE_PWD is not set
+# CONFIG_NSH_DISABLE_REBOOT is not set
+# CONFIG_NSH_DISABLE_RM is not set
+# CONFIG_NSH_DISABLE_RMDIR is not set
+# CONFIG_NSH_DISABLE_SET is not set
+# CONFIG_NSH_DISABLE_SLEEP is not set
+# CONFIG_NSH_DISABLE_SOURCE is not set
+# CONFIG_NSH_DISABLE_TELNETD is not set
+# CONFIG_NSH_DISABLE_TEST is not set
+# CONFIG_NSH_DISABLE_TIME is not set
+# CONFIG_NSH_DISABLE_UMOUNT is not set
+# CONFIG_NSH_DISABLE_UNAME is not set
+# CONFIG_NSH_DISABLE_UNSET is not set
+# CONFIG_NSH_DISABLE_USLEEP is not set
+# CONFIG_NSH_DISABLE_XD is not set
+CONFIG_ARCH="arm"
+CONFIG_ARCH_BOARD="phy6222"
+CONFIG_ARCH_BOARD_PHY6222=y
+CONFIG_ARCH_CHIP="phy62xx"
+CONFIG_ARCH_CHIP_PHY6222=y
+CONFIG_ARCH_CHIP_PHY62XX=y
+CONFIG_ARCH_STACKDUMP=y
+CONFIG_BOARDCTL=y
+CONFIG_BOARDCTL_RESET=y
+CONFIG_BOARD_LATE_INITIALIZE=y
+CONFIG_BUILTIN=y
+CONFIG_CLOCK_MONOTONIC=y
+CONFIG_DEBUG_ASSERTIONS=y
+CONFIG_DEBUG_ERROR=y
+CONFIG_DEBUG_FEATURES=y
+CONFIG_DEBUG_FS=y
+CONFIG_DEBUG_FULLOPT=y
+CONFIG_DEBUG_HARDFAULT_ALERT=y
+CONFIG_DEBUG_LIB=y
+CONFIG_DEBUG_MM=y
+CONFIG_DEBUG_SCHED=y
+CONFIG_DEBUG_SYMBOLS=y
+CONFIG_DEBUG_TIMER=y
+CONFIG_DEBUG_WATCHDOG=y
+CONFIG_DEFAULT_SMALL=y
+CONFIG_DEV_GPIO=y
+CONFIG_EXAMPLES_HELLO=y
+CONFIG_EXAMPLES_TIMER=y
+CONFIG_EXAMPLES_WATCHDOG=y
+CONFIG_FS_AUTOMOUNTER=y
+CONFIG_FS_LITTLEFS=y
+CONFIG_FS_LITTLEFS_BLOCK_FACTOR=1
+CONFIG_FS_PROCFS=y
+CONFIG_FS_TEST=y
+CONFIG_FS_TEST_API=y
+CONFIG_FS_TEST_PERFORMANCE=y
+CONFIG_IDLETHREAD_STACKSIZE=2048
+CONFIG_INIT_ENTRYNAME="init"
+CONFIG_INIT_ENTRYPOINT="nsh_main"
+CONFIG_INIT_STACKSIZE=2560
+CONFIG_INTELHEX_BINARY=y
+CONFIG_IOEXPANDER=y
+CONFIG_IOEXPANDER_MULTIPIN=y
+CONFIG_LIBC_LOCALTIME=y
+CONFIG_MM_CIRCBUF=y
+CONFIG_MM_SMALL=y
+CONFIG_MM_TEST=y
+CONFIG_MTD=y
+CONFIG_NFILE_DESCRIPTORS_PER_BLOCK=6
+CONFIG_NSH_BUILTIN_APPS=y
+CONFIG_NSH_FILEIOSIZE=64
+CONFIG_NUNGET_CHARS=0
+CONFIG_PIPES=y
+CONFIG_PREALLOC_TIMERS=0
+CONFIG_PSEUDOFS_ATTRIBUTES=y
+CONFIG_PSEUDOFS_SOFTLINKS=y
+CONFIG_PTHREAD_CLEANUP=y
+CONFIG_PTHREAD_CLEANUP_STACKSIZE=2
+CONFIG_PTHREAD_MUTEX_TYPES=y
+CONFIG_PTHREAD_SPINLOCKS=y
+CONFIG_PTHREAD_STACK_DEFAULT=3072
+CONFIG_RAM_SIZE=58368
+CONFIG_RAM_START=0x1fff1c00
```

Comment:
the CONFIG_RAM_START and CONFIG_RAM_SIZE should be equal to the value defined in flash.ld.
In flash.ld, the CONFIG_RAM_START = 0x1fff1880 and CONFIG_RAM_SIZE = 59264

Reply:
i will check and fix it.